charts/athens-proxy/values.yaml

```diff
@@ -35,9 +35,6 @@ service:
   ## from within the cluster. To expose externally, consider a "NodePort" or "LoadBalancer" service.
   type: ClusterIP
   ## Further configuration if service is of type "NodePort"
-  nodePort:
```

Comment:
This must stay, but may be commented out as it is only used when `type: NodePort`